pywinauto/uia_defines.py

```diff
@@ -218,6 +218,8 @@ class NoPatternInterfaceError(Exception):
 scroll_large_increment = IUIA().ui_automation_client.ScrollAmount_LargeIncrement
 scroll_small_increment = IUIA().ui_automation_client.ScrollAmount_SmallIncrement
 
+vt_empty = IUIA().ui_automation_client.VARIANT.empty.vt
```

Comment:
These values are defined in win32defines.py.

Reply:
Yep, you are right. But I intentionally didn't want to expose win32defines to any UIA modules. Probably too late, since there are already win32controls and common_controls imported in uia_controls... Honestly, I would move all these win32 hacks that have been added lately in UIA into a separate backend. Something like "hybrid".

Reply:
I think having third backend on Windows is too much. UIA can be optional, but Win32API is used for `Application` object and all other basic things on Windows. So I wouldn't support the idea to create a "hybrid" backend. Making UIA optional / separable (mostly for WinXP) should be enough.

This particular definition is OK to me in both variations. We can leave it as is since it's already merged.